zarr/util.py

```diff
@@ -365,3 +367,142 @@ def __repr__(self):
     def _repr_html_(self):
         items = self.obj.info_items()
         return info_html_report(items)
+
+
+class ZarrGroupTraversal(Traversal):
+
+    def get_children(self, node):
+        return getattr(node, "values", lambda: [])()
+
+    def get_root(self, tree):
+        return tree
+
+    def get_text(self, node):
+        name = node.name.split("/")[-1] or "/"
+        name += "[...]" if hasattr(node, "dtype") else ""
```

Comment:
I like that '[...]' is a concise way to indicate an array. I did wonder if instead people would prefer to see array shape and dtype, which serves to indicate array and also provides some more diagnostics. I'm in two minds. Any thoughts on this?